osquery/core/conversions.cpp

```diff
@@ -316,49 +315,44 @@ bool JSON::valueToBool(const rj::Value& value) {
   return false;
 }
 
-std::string base64Decode(const std::string& encoded) {
-  std::string is;
-  std::stringstream os;
-
-  is = encoded;
-  boost::replace_all(is, "\r\n", "");
-  boost::replace_all(is, "\n", "");
-  size_t size = is.size();
+std::string base64Decode(std::string encoded) {
+  boost::replace_all(encoded, "\r\n", "");
+  boost::replace_all(encoded, "\n", "");
```

Comment:
Up to you, but it would be a bit faster and understandable:
```c++
boost::erase_all(encoded, "\r");
boost::erase_all(encoded, "\n");
```

Reply:
Thanks.
Replaced it with   
 boost::erase_all(encoded, "\r\n");
  boost::erase_all(encoded, "\n");

your method will also remove tabs

Reply:
🤔 Hmm, just a question, why it removes tabs?
I though it removes only `CR  '\r' (carriage ret)` and `LF  '\n' (new line)`.

Reply:
yes, but string with `'\r'` alone is pretty suspicious. Check it out ;) 
```bash
 printf 'ugly truth\rsweet lie\n'
```
But you are right, probably it is not a good idea to change a logic in this PR.